pkg/operators/localmanager/localmanager.go

```diff
@@ -314,6 +314,12 @@ func (l *localManagerTrace) PreGadgetRun() error {
 			},
 		)
 
+		if host {
+			// We need to attach this fake container for gadget which rely only on the
+			// Attacher concept.
+			containers := append(containers, &containercollection.Container{Pid: 1,})
```

Comment:
```suggestion
			containers = append(containers, &containercollection.Container{Pid: 1,})
```

Reply:
After this change it works on my system. 


```bash 
$ go run -exec sudo ./cmd/ig/. trace dns --host 
CONTAINER                         PID        TID        COMM             QR TYPE      QTYPE      NAME                              RCODE              NUMANSW…
                                  309974     309975     isc-worker0000   Q  OUTGOING  A          inspektor-gadget.io.                                 0       
                                  1126       1126       systemd-resolve  Q  HOST      A          inspektor-gadget.io.                                 0       
                                  1126       1126       systemd-resolve  Q  OUTGOING  A          inspektor-gadget.io.                                 0       
                                  1126       1126       systemd-resolve  R  HOST      A          inspektor-gadget.io.              NoError            2       
                                  1126       1126       systemd-resolve  R  OUTGOING  A          inspektor-gadget.io.              NoError            2       
                                  309974     309975     isc-worker0000   R  HOST      A          inspektor-gadget.io.              NoError            2   
```